diff --git a/index.d.ts b/index.d.ts
index a18f70798e3383005dfb2932bf07a2f52c9badff..1e8693c33ef5f4ebbb41fcf921d7cc30f2879da1 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -4,13 +4,13 @@ import { MessageFormatElement } from "@formatjs/icu-messageformat-parser";
 export type TypedMessageDescriptor<V extends MessageValues$1 = {}> = Readonly<MessageDescriptor> & MessageContract<V>;
 export type UntypedMessageDescriptor = MessageDescriptor & UntypedMessageContract;
 export type TypedMessageArguments<V extends MessageValues$1, T = string, TChunk = T> = MessageFormatArguments<V, T, TChunk, [options?: Options]>;
-export declare function defineMessage<V extends MessageValues$1 = {}, const D extends MessageDescriptor = never>(message: D, options?: {
+export declare function defineMessage<V extends MessageValues$1 = {}, const D extends MessageDescriptor = never>(message: D, options: {
   typed: true;
 }): Readonly<D> & Omit<TypedMessageDescriptor<NoInfer<V>>, keyof D>;
 export declare function defineMessage<V extends MessageValues$1 = {}>(message: MessageDescriptor & {
   id: NonNullable<MessageDescriptor["id"]>;
   defaultMessage: string;
-}, options?: {
+}, options: {
   typed: true;
 }): TypedMessageDescriptor<NoInfer<V>> & Readonly<{
   id: NonNullable<MessageDescriptor["id"]>;
@@ -19,7 +19,7 @@ export declare function defineMessage<V extends MessageValues$1 = {}>(message: M
 export declare function defineMessage<V extends MessageValues$1 = {}>(message: MessageDescriptor & {
   id: NonNullable<MessageDescriptor["id"]>;
   defaultMessage: NonNullable<MessageDescriptor["defaultMessage"]>;
-}, options?: {
+}, options: {
   typed: true;
 }): TypedMessageDescriptor<NoInfer<V>> & Readonly<{
   id: NonNullable<MessageDescriptor["id"]>;
@@ -27,36 +27,36 @@ export declare function defineMessage<V extends MessageValues$1 = {}>(message: M
 }>;
 export declare function defineMessage<V extends MessageValues$1 = {}>(message: MessageDescriptor & {
   defaultMessage: string;
-}, options?: {
+}, options: {
   typed: true;
 }): TypedMessageDescriptor<NoInfer<V>> & Readonly<{
   defaultMessage: string;
 }>;
 export declare function defineMessage<V extends MessageValues$1 = {}>(message: MessageDescriptor & {
   defaultMessage: NonNullable<MessageDescriptor["defaultMessage"]>;
-}, options?: {
+}, options: {
   typed: true;
 }): TypedMessageDescriptor<NoInfer<V>> & Readonly<{
   defaultMessage: NonNullable<MessageDescriptor["defaultMessage"]>;
 }>;
 export declare function defineMessage<V extends MessageValues$1 = {}>(message: MessageDescriptor & {
   id: NonNullable<MessageDescriptor["id"]>;
-}, options?: {
+}, options: {
   typed: true;
 }): TypedMessageDescriptor<NoInfer<V>> & Readonly<{
   id: NonNullable<MessageDescriptor["id"]>;
 }>;
-export declare function defineMessage<V extends MessageValues$1 = {}>(message: MessageDescriptor, options?: {
+export declare function defineMessage<V extends MessageValues$1 = {}>(message: MessageDescriptor, options: {
   typed: true;
 }): TypedMessageDescriptor<NoInfer<V>>;
 export declare function defineMessage<T>(message: T): Readonly<T>;
-export declare function defineMessages<V extends Record<string, MessageValues$1> = never, const D extends { [K in keyof NoInfer<V>]: MessageDescriptor; } = never>(messages: [NoInfer<V>] extends [never] ? never : D, options?: {
+export declare function defineMessages<V extends Record<string, MessageValues$1> = never, const D extends { [K in keyof NoInfer<V>]: MessageDescriptor; } = never>(messages: [NoInfer<V>] extends [never] ? never : D, options: {
   typed: true;
 }): { readonly [K in keyof NoInfer<V>]: Readonly<D[K]> & Omit<TypedMessageDescriptor<NoInfer<V[K]>>, keyof D[K]>; };
 export declare function defineMessages<V extends Record<string, MessageValues$1> = never>(messages: [NoInfer<V>] extends [never] ? never : { [K in keyof NoInfer<V>]: MessageDescriptor & {
   id: NonNullable<MessageDescriptor["id"]>;
   defaultMessage: string;
-}; }, options?: {
+}; }, options: {
   typed: true;
 }): { readonly [K in keyof NoInfer<V>]: TypedMessageDescriptor<NoInfer<V[K]>> & Readonly<{
   id: NonNullable<MessageDescriptor["id"]>;
@@ -65,7 +65,7 @@ export declare function defineMessages<V extends Record<string, MessageValues$1>
 export declare function defineMessages<V extends Record<string, MessageValues$1> = never>(messages: [NoInfer<V>] extends [never] ? never : { [K in keyof NoInfer<V>]: MessageDescriptor & {
   id: NonNullable<MessageDescriptor["id"]>;
   defaultMessage: NonNullable<MessageDescriptor["defaultMessage"]>;
-}; }, options?: {
+}; }, options: {
   typed: true;
 }): { readonly [K in keyof NoInfer<V>]: TypedMessageDescriptor<NoInfer<V[K]>> & Readonly<{
   id: NonNullable<MessageDescriptor["id"]>;
@@ -73,29 +73,29 @@ export declare function defineMessages<V extends Record<string, MessageValues$1>
 }>; };
 export declare function defineMessages<V extends Record<string, MessageValues$1> = never>(messages: [NoInfer<V>] extends [never] ? never : { [K in keyof NoInfer<V>]: MessageDescriptor & {
   defaultMessage: string;
-}; }, options?: {
+}; }, options: {
   typed: true;
 }): { readonly [K in keyof NoInfer<V>]: TypedMessageDescriptor<NoInfer<V[K]>> & Readonly<{
   defaultMessage: string;
 }>; };
 export declare function defineMessages<V extends Record<string, MessageValues$1> = never>(messages: [NoInfer<V>] extends [never] ? never : { [K in keyof NoInfer<V>]: MessageDescriptor & {
   defaultMessage: NonNullable<MessageDescriptor["defaultMessage"]>;
-}; }, options?: {
+}; }, options: {
   typed: true;
 }): { readonly [K in keyof NoInfer<V>]: TypedMessageDescriptor<NoInfer<V[K]>> & Readonly<{
   defaultMessage: NonNullable<MessageDescriptor["defaultMessage"]>;
 }>; };
 export declare function defineMessages<V extends Record<string, MessageValues$1> = never>(messages: [NoInfer<V>] extends [never] ? never : { [K in keyof NoInfer<V>]: MessageDescriptor & {
   id: NonNullable<MessageDescriptor["id"]>;
-}; }, options?: {
+}; }, options: {
   typed: true;
 }): { readonly [K in keyof NoInfer<V>]: TypedMessageDescriptor<NoInfer<V[K]>> & Readonly<{
   id: NonNullable<MessageDescriptor["id"]>;
 }>; };
-export declare function defineMessages<V extends Record<string, MessageValues$1> = never>(messages: [NoInfer<V>] extends [never] ? never : { [K in keyof NoInfer<V>]: MessageDescriptor; }, options?: {
+export declare function defineMessages<V extends Record<string, MessageValues$1> = never>(messages: [NoInfer<V>] extends [never] ? never : { [K in keyof NoInfer<V>]: MessageDescriptor; }, options: {
   typed: true;
 }): { readonly [K in keyof NoInfer<V>]: TypedMessageDescriptor<NoInfer<V[K]>>; };
-export declare function defineMessages<const D extends Record<string, MessageDescriptor>>(messages: D, options?: {
+export declare function defineMessages<const D extends Record<string, MessageDescriptor>>(messages: D, options: {
   typed: true;
 }): { readonly [K in keyof D]: Readonly<D[K]> & Omit<TypedMessageDescriptor, keyof D[K]>; };
 export declare function defineMessages<K extends keyof any, T = MessageDescriptor, U extends Record<K, T> = Record<K, T>>(messages: U): { readonly [P in keyof U]: Readonly<U[P]>; };
